server/src/frontend_api.rs

```diff
@@ -568,10 +569,41 @@ security(
 )]
 #[post("/client/metrics")]
 async fn post_frontend_metrics(
+    req: HttpRequest,
     edge_token: EdgeToken,
+    connect_via: Data<ConnectVia>,
```

Comment:
Why are you adding this field? You explicitly set L591 to None, so you don't currently need it.

Alternately, change L591 to `Some(connect_via.0.clone())`

Reply:
register_client_application function requires connect_via reference